cartography/data/jobs/cleanup/opal_aws_cleanup.json

```diff
@@ -0,0 +1,30 @@
+{
+  "statements": [
+    {
+      "query": "MATCH (n:OpalResource) WHERE n.lastupdated <> $UPDATE_TAG WITH n LIMIT $LIMIT_SIZE DETACH DELETE n",
```

Comment:
Let's use the OpalResource model that you created so that we can remove this manual cleanup job. To do this we can introduce a fake "tenant" like in this LastPass PR: https://github.com/cartography-cncf/cartography/pull/1083/files#diff-3b2215e87b2aaf55796fc63a223403d163b97d5bcbb6931d5886802a1f554cd8.